CHANGELOG.md

```diff
@@ -130,6 +130,7 @@ that allows for arbitrary vesting periods.
     * `ValidateSigCountDecorator`: Validate the number of signatures in tx based on app-parameters.
     * `IncrementSequenceDecorator`: Increments the account sequence for each signer to prevent replay attacks.
 * (cli) [\#5223](https://github.com/cosmos/cosmos-sdk/issues/5223) Cosmos Ledger App v2.0.0 is now supported. The changes are backwards compatible and App v1.5.x is still supported.
+* (keys) [\#4941](https://github.com/cosmos/cosmos-sdk/issues/4941) Introduce keybase option to allow overriding the default private key implementation of a key generated through the `keys add` cli command
```

Comment:
This is not enough. Please mention all API breakages (i.e. changes in function signatures) this PR introduces so that app developers know where to look at.

Reply:
There is no breaking changes, the options that can be applied are optional parameters. What specifically were you saying is breaking? Any idea of what you would like me to go more in detail about?

Reply:
Any change in public exported symbols is an API breaking change, e.g. function's signature, return value or even semantic. AFAICS some publicly exported functions have their signatures changed

Reply:
My apologies, assumed since this only makes the key subcommands public and adding optional parameters that it wasn't breaking. Will update now